src/core/or/circuitpadding.c

```diff
@@ -0,0 +1,2370 @@
+/* Copyright (c) 2017 The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+#include <math.h>
+#include "lib/math/fp.h"
+#include "core/or/or.h"
+#include "core/or/circuitpadding.h"
+#include "core/or/circuitlist.h"
+#include "core/or/circuituse.h"
+#include "core/or/relay.h"
+#include "feature/stats/rephist.h"
+#include "feature/nodelist/networkstatus.h"
+
+#include "core/or/channel.h"
+
+#include "lib/time/compat_time.h"
+#include "lib/crypt_ops/crypto_rand.h"
+
+#include "core/or/crypt_path_st.h"
+#include "core/or/circuit_st.h"
+#include "core/or/origin_circuit_st.h"
+#include "feature/nodelist/routerstatus_st.h"
+#include "feature/nodelist/node_st.h"
+#include "core/or/cell_st.h"
+#include "core/or/extend_info_st.h"
+#include "core/crypto/relay_crypto.h"
+#include "feature/nodelist/nodelist.h"
+
+#include "app/config/config.h"
+
+/* XXX: This is a dup of the constant in ./src/lib/time/tvdiff.c.
+ * Should/Do we have a header for time constants like this? */
+#define TOR_USEC_PER_SEC (1000000)
```

Comment:
lib/defs would be a good place to put a header like that.

Reply:
And collect the other TOR_USEC_PER_SEC definitions in the tor codebase.
(We just added another one in 0.3.5.4)

Reply:
Addressed in 9b4c261b2d.

Reply:
Addressed in d8a2e7dcd0.